src/handlers/comment/handlers/multiplier.ts

```diff
@@ -43,7 +56,7 @@ export const multiplier = async (body: string) => {
     }
 
     await upsertWalletMultiplier(username, bountyMultiplier?.toString());
-    return `Updated the multiplier for @${username} successfully!\t New multiplier: ${bountyMultiplier}`;
+    return `Successfully changed the payout multiplier for @${username} to ${bountyMultiplier}. The reason provided is ${reason}.`;
```

Comment:
`reason` is optional so in case it is empty it is better to omit the reason text `The reason provided is ${reason}` or read a default reason from somewhere

Reply:
Done